app/src/main/java/com/expidev/gcmapp/utils/DatabaseOpenHelper.java

```diff
@@ -72,6 +73,8 @@ public void onUpgrade(@NonNull final SQLiteDatabase db, final int oldVersion, fi
                     break;
                 case 9:
                     db.execSQL(Contract.Church.SQL_V9_ALTER_DIRTY);
+                    db.execSQL(Contract.MeasurementDetails.SQL_V9_ALTER_LOCAL);
+                    db.execSQL(Contract.MeasurementDetails.SQL_V9_ALTER_PERSONAL);
```

Comment:
these should be under a `case 10:` since the db would be upgrading from `9` to `10` and would run the migrations for the new version(s) only